x/feegrant/keeper/keeper.go

```diff
@@ -33,8 +32,10 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) GrantFeeAllowance(ctx sdk.Context, grant types.FeeAllowanceGrant) {
 	store := ctx.KVStore(k.storeKey)
 	key := types.FeeAllowanceKey(grant.Granter, grant.Grantee)
-	bz := k.cdc.MustMarshalBinaryBare(grant)
-
+	bz, err := grant.Marshal()
+	if err != nil {
+		panic(fmt.Errorf("failed to encode fee allowance: %w", err))
```

Comment:
Use consistent keywords
```suggestion
		panic(fmt.Errorf("failed to marshal fee allowance: %w", err))
```